tasks/unit_tests/release_tests.py

```diff
@@ -1189,3 +1190,16 @@ def test_changes_integrations_core_warning_branch_out(self, version_mock, print_
         ]
         print_mock.assert_has_calls(calls)
         self.assertEqual(print_mock.call_count, 2)
+
+
+class TestDefaultModules(unittest.TestCase):
+    def test_current_default_modules(self):
```

Comment:
you need a test against the `6.53.x` version of the DEFAULT_MODULES here I guess